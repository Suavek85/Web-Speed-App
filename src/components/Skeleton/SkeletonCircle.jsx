import React from 'react'
import Skeleton, { SkeletonTheme } from "react-loading-skeleton"

export default function SkeletonCircle() {
  return (
    <div style={{marginLeft: '7px'}}>
      <SkeletonTheme color="#bdbdbd" highlightColor="#d8d8d8">
        <Skeleton circle={true} height={46} width={44} />  
      </SkeletonTheme>
    </div>)
}
 
