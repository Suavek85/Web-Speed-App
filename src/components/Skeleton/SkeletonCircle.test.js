import React from 'react'
import { shallow } from 'enzyme'
import SkeletonCircle from './SkeletonCircle'

describe('SkeletonCircle snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<SkeletonCircle />)
    expect(wrapper).toMatchSnapshot()
  })

})