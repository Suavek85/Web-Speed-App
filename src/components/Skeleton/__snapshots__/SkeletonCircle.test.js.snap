// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonCircle snapshot should match the snapshot 1`] = `
<div
  style={
    Object {
      "marginLeft": "7px",
    }
  }
>
  <SkeletonTheme
    color="#bdbdbd"
    highlightColor="#d8d8d8"
  >
    <Skeleton
      circle={true}
      count={1}
      duration={1.2}
      height={46}
      width={44}
      wrapper={null}
    />
  </SkeletonTheme>
</div>
`;
